lib/task/import/csvCheckImportTask.class.php

```diff
@@ -137,6 +137,25 @@ protected function getDbConnection()
 
     protected function setCsvValidatorFilenames($filenameString)
     {
+        // If a directory's provided return an array of file paths
+        if (is_dir($filenameString)) {
+            $files = array_diff(scandir($filenameString), ['..', '.']);
```

Comment:
I'd prefer to see the directory scan logic moved to a separate method for readability and testability, but it's not a blocker.

Reply:
Makes sense!